src/services/zapper.ts

```diff
@@ -26,9 +26,10 @@ export class ZapperService extends Service {
    * @returns list of tokens supported by the zapper protocol.
    */
   async supportedTokens(): Promise<Token[]> {
-    const url = "https://api.zapper.fi/v1/prices";
-    const params = new URLSearchParams({ api_key: this.ctx.zapper });
-    const zapperTokens: ZapperToken[] = await fetch(`${url}?${params}`)
+    const url = "https://api.zapper.fi/v2/prices";
+    const zapperTokens: ZapperToken[] = await fetch(url, {
+      headers: { Authorization: `Basic ${this.ctx.zapper}` },
```

Comment:
a refactor could be having a `this.getFetchDefaultHeaders()` function since it repeats a lot.

Reply:
I was thinking doing something along those lines, or even a `zapperFetch()` wrapper on `fetch()`, didn't do it because it's all in one file, so easy to refactor. Happy to refactor if we think this is too much repetition. Let me know.

Reply:
its a nit really, that is why I approved already. Could be useful if they decide to change how to handle their authentication in the future. Will merge since it can be done later.